### ISSUE TITLE
What I want for Nowhere

### ISSUE THREAD

**Issue by Cruuk** [Tslat/Advent-Of-Ascension#3237]
(If this conflicts with what is actually planned, at least I tried. I only know very little)

Nowhere has high potential. In order for this potential to be fully realized though, Nowhere must not be viewed as a traditional dimension with only 1 way to get to it.

The main problem I have with Nowhere right now is that it calls itself "tierless", when this is currently very false. It is explicitly behind the End, which is technically T2, that's not tierless at all. There is a way to fix this without changing the task, and this plays into what I was saying earlier with not thinking of Nowhere as a traditional dimension.

Nowhere does not need a single "theme", instead its theme should be "the place where any unbreakable structure and content balanced around not being in a sandbox" go. There would also be different ways to get to these unbreakable structures in Nowhere besides the realmstone. You wouldn't be able to get to other parts of Nowhere with non-realmstone methods. The lobby would be exclusive to the realmstone, making it still useful as a shortcut. The activity portals could just lead you to rooms where you select the thing you want to do. E.G. the boss portal leads you to a room with the altars or something.
This will be elaborated on more in the below sections.

The exception to this would be stuff made to be Nowhere-exclusive for obvious reasons.

------------------------

**Bosses**
This is more of a new response to the Nowhere boss thing, remember that I do not know if plans have changed since.

It's overall still a good idea, but the main problem I have with it is that if we think Nowhere as a traditional dimension, it's gating all of them behind 1 dim. This would mess with progression in various ways: If you go through the dims and ignore the bosses, this will make the rewards from most of them useless. They're already optional, which is fine, but this makes them far too much out of the way. And if this change made them all endgame, having a huge amount of endgame bosses all bottlenecked at the end would be very weird. If Nowhere stayed behind the End, all the weapons would have to be t3+, which would make balancing harder and result in choice paralysis. Troll basher axe justifies itself by being OW-accessible, but if it gets placed at the same level as Runic/Guardian sword, it would either have to get powercrept damage to compete or be useless.

It would make bosses very rigid, but I think it can be fixed. Here is an example, once you get Boulder dash and can access Kror, you can use it in 2 different places: taking it to Nowhere, or using it on a Kror altar in Deeplands, which takes you to the same Nowhere arena, and only that arena. Once you finish the fight, exiting the arena if entering from Deeplands would take you back to Deeplands, where exiting the arena if you entered from Nowhere takes you back to the lobby. If entering from Deeplands, you would not be able to access the rest of Nowhere.

From what I see, the point of the Nowhere boss thing was to cut out the problematic aspects of sandbox games and keep it to a controlled environment. This would keep the controlled environment, while keeping progression more smooth. Doing it from a regular dim has the upside of fighting the boss without having to access Nowhere realmstone, while Nowhere lobby would be more of an alternative shortcut that would reduce travel time via the boss portal in the lobby. Boss rewards wouldn't have to get drastic balance changes, and Nowhere can actually be called "tierless" since pieces of it would be accessible in different ways.

Another part of the Nowhere boss thing are "varying arenas". I presume it means that boss arenas would be random. I'm not sure if I would like that. What if a boss acts in a way that makes it easy to cheese in 1 arena, but not the other. It would be un-ideal RNG. Not every boss needs some unique ultra-detailed arena, but even if they use the same one, I would prefer consistency in the arenas over anything else.

----------------------------

**Dungeon**

Have you ever heard of Pikmin 2 and Pikmin 3? I know it's a very random mention, but you'll understand why. In the main modes, the gameplay is slower-paced and you deal with environmental hazards. But it also has a "challenge mode", which are missions that take those same hazards and put them in a more isolated, shorter, replayable arcade-y environment with simple objectives.

This is what I want the Nowhere dungeons to be. Where the regular dims focus more on ambience and being a bit livable, Nowhere would take stuff like mobs from those dims, and utilizing their mechanics in a more intensified isolated environment with arcade-y difficulty. I like arcade-y difficulty. Obviously mobs would not be able to access their normal world tables in Nowhere.

What I define as "arcade-y difficulty" is something that is hard and needs practice, but is easily replayable. Remember that Nowhere has keepInventory **enabled**. Doing stuff like having just 1 dungeon exclusive to Nowhere with just the former Immortallis mobs would be lost potential.

You can still use the former Immortallis mobs, I just think using them and other mobs would be ideal. With this format, you can have multiple dungeons, and do stuff like have a Shyre-centric dungeon that also requires an item from Shyre to enter, or have dungeons with mobs from different dims. You can also do something similar to the idea I had for the bosses, like having ways to access these dungeons from other places that aren't the Nowhere realmstone.

I'm not saying to get overly ambitious, these are examples to emphasize the flexibility of what you can do.

-----------------------------

Can't really think of anything good for specifically parkour/utility, I will leave that to you. Other than keep in mind the thing I've been saying throughout this:

To summarize, Nowhere shouldn't just be a traditional dimension with only 1 way to access it, but rather a tool used when we want any unbreakable structure-related content balanced around keepInventory. It would allow for a lot of things

**Comment by Tslat** [comment]
All already planned
